lib/system/allocator.cpp

```diff
@@ -7,15 +7,8 @@
 #include "common/defines.h"
 #include "common/errcode.h"
 
-#include <cassert>
-#include <cctype>
```

Comment:
Why do you remove lots of headers which are used in this file.

Reply:
These headers are not used.